bootstrap/config/crd/kustomization.yaml

```diff
@@ -1,5 +1,6 @@
 commonLabels:
-  cluster.x-k8s.io/v1beta1: v1alpha1
+  cluster.x-k8s.io/v1alpha1: v1alpha1
```

Comment:
I think these 2 can be replaced with:

```
cluster.x-k8s.io/v1beta1: v1alpha1_v1beta1
```

This is mapping the v1beta1 CAPI version to the CAPRKE2 version v1alpha1 and v1beta1.

Reply:
Thanks, it makes sense. I was looking at how upstream projects do this and I realize now that I didn't understand how it works.